phantom-sbt/src/main/scala/com/websudos/phantom/sbt/SbtPlugin.scala

```diff
@@ -0,0 +1,151 @@
+ * notice, this list of conditions and the following disclaimer in the
+ * documentation and/or other materials provided with the distribution.
+ *
+ * - Explicit consent must be obtained from the copyright owner, Websudos Limited before any redistribution is made.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.websudos.phantom.sbt
+
+import java.util.concurrent.atomic.AtomicBoolean
+
+import org.cassandraunit.utils.EmbeddedCassandraServerHelper
+import sbt.Keys._
+import sbt._
+import sbt.plugins.JvmPlugin
+
+import scala.concurrent.blocking
+import scala.util.Try
+import scala.util.control.NonFatal
+
+/**
+  * sbt plugin for starting Cassandra in embedded mode before running
+  * the tests.
+  *
+  * First the plugin must be included in your `plugins.sbt`:
+  *
+  * {{{
+  *   addSbtPlugin("com.websudos" %% "phantom-sbt" % phantomVersion)
+  * }}}
+  * The plugin does the following
+  * things:
+  *
+  * - Automatically starts Cassandra in embedded mode whenever the test task is run
+  * - Forces the tests for the projects that include the settings to always run in a
+  *   forked JVM as this is the only way to make parallel tests using phantom work.
+  *   (This is not caused by the implementation of this plugin or the new connector
+  *   or zookeeper artifacts, this is caused by implementation details in the official
+  *   `phantom-dsl` artifact, mainly the use of reflection which is not thread-safe
+  *   in Scala 2.10)
+  *
+  * If you want to specify a custom Cassandra configuration,
+  * you can do that with a setting:
+  *
+  * {{{
+  * phantomCassandraConfig := baseDirectory.value / "config" / "cassandra.yaml"
+  * }}}
+  */
+object PhantomSbtPlugin extends AutoPlugin {
+
+  override def requires: Plugins = sbt.plugins.JvmPlugin
+
+  /**
+    * Keys for all settings of this plugin.
+    */
+  object autoImport {
+
+    val phantomStartEmbeddedCassandra = taskKey[Unit]("Starts embedded Cassandra")
+    val phantomCleanupEmbeddedCassandra = taskKey[Unit]("Clean up embedded Cassandra by dropping all of its keyspaces")
+    val phantomCassandraConfig = settingKey[Option[File]]("YAML file for Cassandra configuration")
+  }
+
+  import autoImport._
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/flavian/phantom/file/2526339477/issues/source?bid=2847375&fileBranchId=3281943#l84)